### PR TITLE
app-benchmarks/siege: bump to 4.0.9

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

**Comment by thesamesam** [comment]
Ideally don't 'git mv' so we can easily revert commits. The exception is if the current version is critically broken, but even then...

**Comment by scardracs** [comment]
> Ideally don't 'git mv' so we can easily revert commits. The exception is if the current version is critically broken, but even then...

Ok :) I'm adding back 4.0.7 too

File: app-benchmarks/siege/siege-4.0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	local myconf=(
+		$(use_with ssl ssl "${EPREFIX}/usr")
+	)
+	econf "${myconf[@]}"
+}
+
+src_install() {
+	emake DESTDIR="${ED}" install
+
+	dodoc AUTHORS ChangeLog INSTALL README* doc/siegerc doc/urls.txt
```
Comment (juippis) [inline app-benchmarks/siege/siege-4.0.9.ebuild:44]: Ideally you'd define these as a global `DOCS` array, so you could just...

Comment (juippis) [inline app-benchmarks/siege/siege-4.0.9.ebuild:44]: Call `default` here. This only seems to install *README.md*, so no globbing needed.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**